src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -55,24 +55,27 @@
   <!-- ********************************************************** -->
   <!-- Sonar C#/VB analyzers -->
   <!-- ********************************************************** -->
+  <!-- We assume that the package restore has happened so the files are already on disc -->
   <Import Project="..\EmbeddedSonarAnalyzer.props" />
   <ItemGroup Label="Embedded SonarLint analyzers">
     <!-- Reference the packages we want to import, but don't include them as analyzers for this project -->
     <PackageReference Include="SonarAnalyzer.CSharp" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
     <PackageReference Include="SonarAnalyzer.VisualBasic" Version="$(EmbeddedSonarAnalyzerVersion)" ExcludeAssets="all" PrivateAssets="all" />
 
-    <!-- Select all of the analyzer binaries (we assume that the package restore has happened so the files are already on disc) -->
-    <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
-    <SonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.VisualBasic\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
+    <!-- Include all of the analyzer binaries including third-party binaries in the VSIX -->
+    <AllSonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.CSharp\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
+    <AllSonarAnalyzerAssemblies Include="$(NuGetPackageRoot)SonarAnalyzer.VisualBasic\$(EmbeddedSonarAnalyzerVersion)\analyzers\*.dll" />
+    <VSIXSourceItem Include="@(AllSonarAnalyzerAssemblies)" />
 
-    <!-- We need to reference the core analyzer assembly since we use one of the types. -->
+    <!-- Make sure the Sonar.* analyzer assemblies we want to include in the AsmRef check to the output directory. -->
+    <!-- We don't want to copy third-party assemblies from the analyzers in case there are version conflicts e.g. NewtonSoft.Json.dll -->
```

Comment:
If we are only copying the Sonar* assemblies for the AsmRef, then the Newtonsoft.Json.dll appears in the asmref report because it's used by Sonar*?

Reply:
Not quite. SLVS used it anyway so it was always in the report. Now it is listed multiple times, but with different versions:

```
---
Assembly: 'SonarAnalyzer.CSharp, Version=8.26.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
Relative path: 'SonarAnalyzer.CSharp.dll'

Referenced assemblies:
...
- 'Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
...


Assembly: 'SonarLint.2019, Version=4.37.0.0, Culture=neutral, PublicKeyToken=null'
Relative path: 'SonarLint.2019.dll'

Referenced assemblies:
...
- 'Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
...
```